signatures/helpers/arguments_helpers.go

```diff
@@ -39,6 +39,20 @@ func GetTraceeStringArgumentByName(event trace.Event, argName string) (string, e
 	return "", fmt.Errorf("can't convert argument %v to string", argName)
 }
 
+// GetTraceeBoolArgumentByName gets the argument matching the "argName" given from the event "argv" field, casted as bool.
```

Comment:
nit: I think it would be better, instead of expanding these helpers to reduce them to one `GetArgValueByName[T]`.
And since this was added in the signatures package but it's intended use was in the proctree, maybe the above function should be given also in the `events` package (similar to how we have `GetArg` in `parse_args.go` (which i'm not sure is the correct place for actually)).